syntax = "proto3";

package cpu_user_usage;

service GetCpuUserUsageService {
    rpc GetCpuUserUsage (GetCpuUserUsageRequest) returns (GetCpuUserUsageResponse);
}

message GetCpuUserUsageRequest {
    int64 date_from = 1;
    int64 date_to = 2;
}

message CpuUsage {
    string cpu = 1;
    double avg_usage = 2;
    double max_usage = 3;
    double min_usage = 4;
  }
  
message GetCpuUserUsageResponse {
    repeated CpuUsage usages = 1;
  }
